packages/frameworks-remix/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@auth/remix",
+  "description": "Authentication for Remix.",
+  "version": "0.1.0",
+  "type": "module",
+  "files": [
+    "client.*",
+    "index.*",
+    "src"
+  ],
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "import": "./index.js"
+    },
+    "./client": {
+      "types": "./client.d.ts",
+      "import": "./client.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "tsup --config ./tsup.config.js && node scripts/postbuild",
+    "patch": "npm version patch --no-git-tag-version",
+    "clean": "rm -rf client.* index.*"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@auth/core": "workspace:*",
+    "@types/node": "^18.7.14",
+    "next-auth": "workspace:*",
+    "tsup": "^6.5.0",
```

Comment:
let's not add another tool. we already have `tsc`, let's remove this

Reply:
@balazsorban44 

Hmm I'll make the change but just fyi I based it off

https://github.com/nextauthjs/next-auth/tree/main/packages/frameworks-solid-start

Reply:
thanks for the heads up, we should do the same there too. 👍

Reply:
solid start is now using tsc instead of tsup aswell

https://github.com/nextauthjs/next-auth/pull/6276/commits/cad4b59d3cc3c753956bee30f76a75ad45f9e7fa